### ISSUE TITLE
ctc_loss complains  that Labels length is zero

### ISSUE THREAD

**Issue by breadbread1984** [tensorflow/tensorflow#18288]
OS Platform and Distribution ubuntu 16.04
TensorFlow installed from pip 
TensorFlow version 1.7
Bazel version N/A
CUDA/cuDNN version 9.0 7.0
GPU model and memory N/A
Exact command to reproduce N/A

I want to use ctc_loss to do gesture recognition. But I keep getting error message that Labels length is zero at batch 0. I can't figure out the problem. So come here for help.
The data I feed into ctc_loss are listed in the following table

```
tensname                    type                      dimension
labels                  sparsetensor         batch_size(5) x label_length(1)
sequence_predicts         tensor                  time_steps(10) x batch_size(5) x num_classes(26)
sequence_lengths          tensor                  batch_size(5)
```

I create ctc_loss with the following code

```python
loss = tf.nn.ctc_loss(labels,sequence_predicts,sequence_lengths,time_major = True);
```

I print all input tensors with the following code
```python
denselabels = tf.sparse_to_dense(labels.indices,labels.dense_shape,labels.values);
denselabels = tf.Print(denselabels,[denselabels],message = "labels = ", summarize = 20, name = "denselabels");
sequence_predicts = tf.Print(sequence_predicts,[sequence_predicts],message = "sequence_predicts = ", summarize = 20, name = "sequence_predicts");
sequence_lengths = tf.Print(sequence_lengths,[sequence_lengths],message = "sequence_lengths = ", summarize = 200, name = "sequence_lengths");
```

The output on the console is 
```bash
sequence_lengths = [10 10 10 10 10]
labels = [[1][2][1][0][0]]
sequence_predicts = [[[0.0183351934 -0.0540067367 0.0173245296 0.10416019 -0.06072326 -0.0563994423 0.00676568877 -0.046640303 -0.0107666068 0.0554158054 0.0658506081 0.0124566173 0.0133351646 0.0134876268 -0.0453479 -0.058809232 0.0160786062 -0.0285613891 -0.105710872 0.0354634598]]...]
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by Yue-Rain** [comment]
hello!
i met the same question during the training of speech recognition.

> OS Platform : ubuntu 16.04

> TensorFlow installed from pip

> TensorFlow version 1.6

here is my label format of the speech:

> utterance_id dialect_id

> 20170809_zj L
when i ran the training code ,the errors occurred:**Labels length is zero in batch 0**(here batch = 10)
what confused me is that the length of the label is 1 ,not 0.why the error happens?
or where did i mistaken?
Thank you!
